intro/scipy/index.rst

```diff
@@ -610,64 +610,63 @@ Statistics and random numbers: :mod:`scipy.stats`
     >>> np.random.seed(0)
 
 
-The module :mod:`scipy.stats` contains statistical tools and probabilistic
-descriptions of random processes. Random number generators for various
-random process can be found in :mod:`numpy.random`.
-
-Distributions: histogram and probability density function
-..........................................................
-
-Given observations of a random process, their histogram is an estimator of
-the random process's PDF (probability density function): ::
-
-    >>> samples = np.random.normal(size=1000)
-    >>> bins = np.arange(-4, 5)
-    >>> bins
-    array([-4, -3, -2, -1,  0,  1,  2,  3,  4])
-    >>> histogram = np.histogram(samples, bins=bins, density=True)[0]
-    >>> bins = 0.5*(bins[1:] + bins[:-1])
-    >>> bins
-    array([-3.5, -2.5, -1.5, -0.5,  0.5,  1.5,  2.5,  3.5])
-    >>> pdf = sp.stats.norm.pdf(bins)  # norm is a distribution object
-
-    >>> plt.plot(bins, histogram) # doctest: +ELLIPSIS
-    [<matplotlib.lines.Line2D object at ...>]
-    >>> plt.plot(bins, pdf) # doctest: +ELLIPSIS
-    [<matplotlib.lines.Line2D object at ...>]
+:mod:`scipy.stats` contains fundamental tools for statistics in Python.
+
+Statistical Distributions
+.........................
+
+Consider a random variable distributed according to the standard normal.
+We draw a sample consisting of 100000 observations from the random variable.
+The normalized histogram of the sample is an estimator of the random
+variable's probability density function (PDF)::
+
+    >>> dist = sp.stats.norm(loc=0, scale=1)  # standard normal distribution
+    >>> sample = dist.rvs(size=100000)  # "random variate sample"
+    >>> plt.hist(sample, bins=50, density=True, label='normalized histogram')  # doctest: +SKIP
+    >>> x = np.linspace(-5, 5)
+    >>> plt.plot(x, dist.pdf(x), label='PDF')  # doctest: +SKIP
+    >>> plt.legend()
 
 .. image:: auto_examples/images/sphx_glr_plot_normal_distribution_001.png
     :target: auto_examples/plot_normal_distribution.html
     :scale: 70
 
-.. sidebar:: **The distribution objects**
+.. sidebar:: **Distribution objects and frozen distributions**
 
-   :class:`scipy.stats.norm` is a distribution object: each distribution
-   in :mod:`scipy.stats` is represented as an object. Here it's the
-   normal distribution, and it comes with a PDF, a CDF, and much more.
+   Each of the 100+ :mod:`scipy.stats` distribution families is represented by an
+   *object* with a `__call__` method. Here, we call the :class:`scipy.stats.norm`
+   object to specify its location and scale, and it returns a *frozen*
+   distribution: a particular element of a distribution family with all
+   parameters fixed. The frozen distribution object has methods to compute
+   essential functions of the particular distribution.
```

Comment:
The new distribution infrastructure will represent distribution families with classes, and particular distributions will be instances of the class. In the meantime, we should be explicit about what is going on because it is commonly misunderstood.